test/openssl/test_pkey.rb

```diff
@@ -90,6 +90,7 @@ def test_hmac_sign_verify
   def test_ed25519
     # Ed25519 is not FIPS-approved.
     omit_on_fips
+    omit "Ed25519 not supported" unless openssl?(1, 1, 1)
```

Comment:
```suggestion
    omit "Ed25519 not supported" unless openssl?(1, 1, 1) || libressl?(3, 7, 0)
```

Non-`ASN1_item_sign()` Ed25519 usage is supported in LibreSSL >= 3.7.0.

Reply:
Yeah, I had a questions about that. The original code didn't guard against that, but it makes sense for it to be consistent across `test_x509*` tests. I'll update it.

Reply:
Looking at https://github.com/libressl/portable/blob/1a3e756a757da607621949b43f97fb79b6fb31ae/ChangeLog#L352 did you mean `libressl?(3, 8, 1)`?

Reply:
That changelog entry is for `ASN1_item_sign()`. Ed25519 in `EVP_PKEY_sign()` was added in 3.7.0. https://github.com/libressl/portable/blob/1a3e756a757da607621949b43f97fb79b6fb31ae/ChangeLog#L591-L592